### DISCUSSION TITLE
Cannot boot up nodemon alongside with vue-cli-service

### DISCUSSION THREAD

**Discussion by handhikadj** [codesandbox/codesandbox-client#4344] [General]
https://codesandbox.io/s/reusable-bootstrap-vue-datatable-sdftd

I've added the dev script inside package.json but still can't get it booted up

**Comment by garethx** [comment]
For performance reasons we ignore any specified scripts in client sandboxes, instead using a default script. If you need to control the scripts, then we recommend using a container sandbox.
